Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -112,8 +111,7 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 
 	double GetClientRTT() const { return AveragedClientRTTSeconds; }
 	double GetClientUpdateTimeDelta() const { return AveragedClientUpdateTimeDeltaSeconds; }
-	double GetPlayersConnected() const { return ActivePlayers; }
-	double GetTotalAuthoritativePlayers() const;
+	double GetPlayersConnected() const { return (double)ActivePlayers; }
```

Comment:
I prefer explicit c++ casts were used instead of c-style casting ... e.g. static_cast<double>(ActivePlayers) .. though is it even necessary? implicit conversion from int32 to double should happen here.